docs/.vuepress/theme/styles/index.styl

```
@@ -177,6 +177,7 @@ a {
 table {
   width: 100%;
   display: table;
+  word-break:break-all;
```

```suggestion
  word-break: break-word;
```
Now it cut the text in random places, I suggest using `break-word`. It breaks words in a more natural way.

---

Good idea, thank you